docs/README.md

```diff
@@ -0,0 +1,214 @@
+
+![Automation Settings button](./assets/gear-button.png)
+
+4. In the `Automation Settings` popup:
+- In `Language` section, choose `NodeJS`.
+- In `App Type` section, choose `Hybrid/Native from Url`.
+- In `Application Url` section, put `https://appium.github.io/appium/assets/ApiDemos-debug.apk`.
+
+![Example Android DesiredCaps](./assets/desired-caps-example-android.png)
+
+5. Add these environment variables to your project configuration in CircleCI with values of corresponding variables taken from the collected desired capabilities as shown in the table below
+
+| CircleCI Environment Variables      | Desired Capabilities Variables | Default Value                                             |
+|-------------------------------------|--------------------------------|-----------------------------------------------------------|
+| KOBITON_DEVICE_PLATFORM_NAME        | platformName                   | Android                                                   |
+| KOBITON_DEVICE_NAME                 | deviceName                     | Galaxy*                                                   |
+| KOBITON_DEVICE_PLATFORM_VERSION     | platformVersion                | *No*                                                      |
+| KOBITON_SESSION_DEVICE_ORIENTATION  | deviceOrientation              | portrait                                                  |
+| KOBITON_SESSION_CAPTURE_SCREENSHOTS | captureScreenshots             | true                                                      |
+| KOBITON_SESSION_DEVICE_GROUP        | deviceGroup                    | KOBITON                                                   |
+| KOBITON_ORGANIZATION_GROUP_ID       | groupId                        | *No*                                                      |
+| KOBITON_SESSION_APPLICATION_URL     | app                            | https://appium.github.io/appium/assets/ApiDemos-debug.apk |
+
+For example, if desired capabilities for executing automation test of the provided demo Android application on Pixel 2 XL running Android 8.1.0:
+
+```javascript
+var desiredCaps = { 
+  sessionName:        'Sample Automation Test Session',
+  sessionDescription: 'This is a sample automation test session', 
+  deviceOrientation:  'portrait',  
+  captureScreenshots: false, 
+  app:                'https://appium.github.io/appium/assets/ApiDemos-debug.apk',
+  deviceGroup:        'KOBITON', 
+  groupId:            100,
+  deviceName:         'Pixel 2 XL',
+  platformVersion:    '8.1.0',
+  platformName:       'Android' 
+}
+```
+
+The environment variables representing above desired capabilities should look like this:
+
+![Desired Caps To ENV Example](./assets/desired-caps-circleci-env.png)
+
+> More information about Desired Capabilities and its parameters can be found in https://docs.kobiton.com/automation-testing/desired-capabilities-usage/
+
+#### 2.3. CircleCI configuration file
+The provided CircleCI configuration file we will be using looks like this:
```

Comment:
"The provided CircleCI configuration file we will be using looks like this:" should change the sentence structure => suggest: "The provided CircleCI configuration file that will be used looks like this"